templates.js

```diff
@@ -170,5 +170,52 @@ collectHeatmapClicks(function (targetPath) {
         { id: 'interval', type: 'number', displayName: 'Time interval', description: 'Number of milliseconds to prevent click events spam', default: 100 },
       ],
     },
+    {
+      id: 'formTimingTracking',
+      name: 'Form timing tracking',
+      description: `
+        This script will track events of focus, blur and change on <input> and <select> fields of <form> elements.
+        Script will calculate the time spent on given field and submit value in seconds as Custom Event value (e.g. 2.345).
+        <form> element on your website will be automatically detected.
+      `,
+      template: `
+${fs.readFileSync(path.join(__dirname, 'build/formTimingTracking.js'), { encoding: 'utf-8' })}
+
+formTimingTracking(function (fieldInteractionData) {
+  window._paq.push([
+                  'trackEvent',
+                  '{{eventCategoryPrefix}}'+fieldInteractionData.formName,
+                  fieldInteractionData.fieldName,
+                  fieldInteractionData.leaveType,
+                  fieldInteractionData.timeSpent/1000
+                  ])
+}, {
+  formNameAttribute: '{{formNameAttribute}}',
+  fieldNameAttribute: '{{fieldNameAttribute}}',
+});
+      `,
+      arguments: [
+        { id: 'eventCategoryPrefix',
+          type: 'text',
+          displayName: 'Event category prefix',
+          description: 'Prefix that will preceed form name in Custom Event Category of the event that you will track',
+          default: 'Form timing: '
+        },
+        { id: 'formNameAttribute',
+          type: 'text',
+          displayName: 'Form name attribute',
+          description: 'Attribute of the <form> element that will be used to identify the form',
+          default: 'id',
+          choices: ['id','name','label','placeholder']
+        },
+        { id: 'fieldNameAttribute',
+          type: 'text',
+          displayName: 'Input label attribute',
```

Comment:
Because of we define attribute in fly we may omit "name/label", just treat it as "Form/Input attribute"

Reply:
I want this to be consistent with names presented to the user, so I left it as is, but I improved the descriptions to be consistent with underlying names.